hw/ip/prim/dv/prim_lfsr/prim_lfsr_sim_cfg.hjson

```diff
@@ -0,0 +1,60 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  // Name of the sim cfg - typically same as the name of the DUT.
+  name: prim_lfsr
+
+  // Top level dut name (sv module).
+  dut: prim_lfsr
+
+  // Top level testbench name (sv module).
+  tb: prim_lfsr_tb
+
+  // Simulator used to sign off this block
+  tool: vcs
+
+  // Fusesoc core file used for building the file list.
+  fusesoc_core: lowrisc:dv:prim_lfsr_sim:0.1
+
+  // Testplan hjson file.
+  testplan: "{proj_root}/hw/ip/prim/dv/prim_lfsr/prim_lfsr_testplan.hjson"
+
+  // Import additional common sim cfg files.
+  import_cfgs: ["{proj_root}/hw/dv/data/common_sim_cfg.hjson"]
+
+  // Add additional tops for simulation.
+  sim_tops: ["-top prim_lfsr_bind"]
+
+  // Default iterations for all tests - each test entry can override this.
+  reseed: 50
+
+  // Add excl files to tool_srcs so that it gets copied over to the scratch area.
+  //tool_srcs: ["{proj_root}/hw/ip/prim/dv/prim_lfsr/prim_lfsr_cov_excl.el"]
+  tool_srcs: []
+
+  // Refer to the excl file with vcs_cov_excl_files var with the rel path from tool_srcs_dir
+  // so the VCS can find it.
+  //vcs_cov_excl_files: ["{tool_srcs_dir}/prim_lfsr_cov_excl.el"]
+  vcs_cov_excl_files: []
+
+  // Default UVM test and seq class name.
+  uvm_test: ""
+  uvm_test_seq: ""
+
+  // List of test specifications.
+  tests: [
+    {
+      name: prim_lfsr_sanity
+    }
+  ]
+
+  // List of regressions.
+  regressions: [
+    {
+      name: sanity
+      tests: ["prim_lfsr_sanity"]
```

Comment:
Do we want to run this in the private CI?

Reply:
Hmm, I'm not sure that is necessary, @msfschaffner what do you think?

Reply:
The private CI timed out because of this test. Let's remove it and revisit later.

Reply:
sgtm.

Reply:
@sriyerg the private CI is still failing when I remove the `regressions:` field, are you able to see what is the cause?

Reply:
if you want to run this as a quick sanity test, I would suggest to reduce the maximum LFSR size to be tested to ~16bit. the simulation time grows exponentially with bit width, and at the moment the maximum size is set to 32bit which lasts several hours.

https://github.com/lowRISC/opentitan/blob/5a689e8d4c368ffccad34409bf65b755648a7b57/hw/ip/prim/dv/prim_lfsr/tb/prim_lfsr_tb.sv#L26-L30

Reply:
good idea, limiting the lfsr length for sanity testing sounds fine to me

Reply:
just a note here that 32bit might even be a bit long for a nightly run, depending on your runtime target for those regressions...